libraries/AP_HAL_ChibiOS/hwdef/PodmanH7/defaults.parm

```diff
@@ -0,0 +1,12 @@
+# Now setup the default battery pins driver analog pins and default
+
+NTF_DISPLAY_TYPE 1
+
+FRAME_CLASS	1
```

Comment:
I don't think you want to set these - these are specific to copter and this controller could be used on either plane or copter